README.md

```diff
@@ -42,16 +72,36 @@ To install the plugin, run:
   - `status <CR>` : Print the phase and conditions of CRs of a specific type, such as `cephobjectstore`, `cephfilesystem`, etc
   - `purge-osd <osd-id> [--force]` : Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.
 
-- `debug` : [Debug a deployment](#debug-mode)  by scaling it down and creating a debug copy. This is supported for mons and OSDs only
-  - `start  <deployment-name> `
+- `debug` : [Debug a deployment](./docs/debug.md)  by scaling it down and creating a debug copy. This is supported for mons and OSDs only
+  - `start  <deployment-name>`
     `[--alternate-image <alternate-image>]` : Start debugging a deployment with an optional alternative ceph container image
   - `stop  <deployment-name>` : Stop debugging a deployment
 
 - `help` : Output help text
 
-## Examples
+## Documentation
+
+Visit docs below for complete details about each command and their flags uses.
+
+| Uses | Command | Args |
+| :--: |:-------:| :---:|
+| [Running ceph commands](docs/ceph.md) | `kubectl rook-ceph ceph <args>` | `<any ceph supported args>`|
```

Comment:
Instead of a table, how about a bullet list of the pages? The table repeats a lot of details that were already shown above in the usage. As a table of contents, they just need links to the other pages, then they can find the details of the commands and output on those pages.

Reply:
changed to bullet list.